python/src/aiconfig/editor/server/server_utils.py

```diff
@@ -217,6 +219,9 @@ def init_server_state(app: Flask, edit_config: EditServerConfig) -> Result[None,
                 return Err(f"Failed to load AIConfig from {edit_config.aiconfig_path}: {e}")
     else:
         aiconfig_runtime = AIConfigRuntime.create()  # type: ignore
+        first_supported_model: str = ModelParserRegistry.parser_ids()[0]
+        aiconfig_runtime.add_prompt("prompt_1", Prompt(name="prompt_1", input="", metadata=PromptMetadata(model=first_supported_model)))
+
```

Comment:
This file, and parser_ids return type are the only real changes here, the rest are from black auto-formatting